addon/helpers/reverse.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+const {
+  set,
+  Helper,
+  observer,
+  A: emberArray,
+  isArray
+} = Ember;
+
+export default Helper.extend({
+  compute([array]) {
+    if (!isArray(array)) {
+      return [];
```

Comment:
Can we return `[array]`, and update our unit test? This might help with chaining.